docs/sources/mimir/release-notes/v2.6.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Grafana Mimir version 2.6 release notes"
+menuTitle: "V2.6 release notes"
+description: "Release notes for Grafana Mimir version 2.6"
+weight: 300
+---
+
+# Grafana Mimir version 2.6 release notes
+
+Grafana Labs is excited to announce version 2.6 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Lower memory usage in store-gateway by streaming series results**
+  The store-gateway can now stream results back to the querier instead of buffering them. This is expected to greatly reduce peak memory consumption while keeping latency the same. This is still an **experimental feature** and is subject to a changing API and instability.
```

Comment:
We can smooth the messaging here. API won't change and we're not aware of any issue. We'll run in all zones once 2.6 stable will be published.

```suggestion
  The store-gateway can now stream results back to the querier instead of buffering them. This is expected to greatly reduce peak memory consumption while keeping latency the same. This is still an **experimental feature** but Grafana Labs is already running it in production and there's no known issue. This feature can be enabled setting the `-blocks-storage.bucket-store.batch-series-size` configuration option (if you want to try it out, we recommend you setting to 5000).
```